src/constants.js

```diff
@@ -59,6 +59,9 @@ export const CONFIG_DIRECTORY = getDirectory('config');
 export const LINK_REGISTRY_DIRECTORY = path.join(CONFIG_DIRECTORY, 'link');
 export const GLOBAL_MODULE_DIRECTORY = path.join(CONFIG_DIRECTORY, 'global');
 
+export const POSIX_GLOBAL_PREFIX = '/usr/local';
```

Comment:
@BYK Implementing NPM's behavior of using parent directories of the node executable will come with bugs, I've described this in #3458 and https://github.com/yarnpkg/yarn/issues/2064#issuecomment-303823304.

[Daniel](https://github.com/yarnpkg/yarn/issues/2064#issuecomment-303592899) also said something similar:

> The issue is that Yarn symlinks the executables into the wrong directory (the Node.js directory rather than somewhere reasonable). It doesn't really make sense to use the Node.js directory for these scripts, and on many systems that directory is not even writable by regular users.